### PR TITLE
Fix github workflow event

### PR DESCRIPTION
**Pull request by dorsha**
## Description

Fix GitHub workflow event to it will work on forks.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: push
+on:
+  pull_request:
+  push:
```
Comment (nirgur) [inline .github/workflows/ci.yml:5]: Cool 👍 
Why do we need to run the CI on push to main?

**Comment by nirgur** [comment]
Thanks!

**Comment by nirgur** [comment]
@dorsha can you remove the coverage from the list of checks?

**Comment by dorsha** [comment]
> @dorsha can you remove the coverage from the list of checks?

Done

**Pull request merged**